### DISCUSSION TITLE
Is the API down?

### DISCUSSION THREAD

**Discussion by JoseMiguelVazquez** [ManuCastrillonM/disney-api#31] [General]
Hello!, just wondering if the API is down or it has changed in some way

**Comment by ManuCastrillonM** [comment]
Hello!
This project is completely free and I work on it in my free time. However, the infrastructure required to keep it running has a monthly cost which I have been covering for the past three years. In order to keep the API available, I enabled a "[support](https://disneyapi.dev/support-us/)" option on the website where you can contribute to help cover the monthly expenses.

The cost is not very high, but it adds up over time, and the availability of the service will depend on the support we receive. I appreciate your use of the API and hope that you can contribute to keeping it available.
